ssh/tailssh/tailssh_integration_test.go

```diff
@@ -103,10 +112,15 @@ func TestIntegrationSSH(t *testing.T) {
 		{
 			cmd:  "pwd",
 			want: []string{homeDir},
+			skip: runtime.GOOS != "linux" || !suPresent || !loginPresent,
```

Comment:
you need both to be present? or just one?

Reply:
On some distributions, we need both.

Reply:
should we add different tests for different repos then?

Reply:
The Makefile runs tests on both Fedora and Ubuntu, so I think the answer to your question is "we already do".